flink/cr-examples/flinkdeployment/openshift/production-application-cluster.yaml

```diff
@@ -58,12 +58,23 @@ spec:
               mountPath: /opt/flink/log
             - name: flink-volume
               mountPath: /opt/flink/volume
+            - name: flow-volume
+              mountPath: /opt/flink/ibm-flow/flow.json
+              subPath: flow.json
+              readOnly: true
+            - name: flow-volume
+              mountPath: /opt/flink/ibm-flow/config.yaml
+              subPath: config.yaml
+              readOnly: true
       volumes:
         - name: flink-logs
           emptyDir: {}
         - name: flink-volume
           persistentVolumeClaim:
             claimName: ibm-flink-pvc
+        - name: flow-volume
+          secret:
+            secretName: application-cluster-prod
```

Comment:
The new secret name mentioned here is in line with the current [EventProcessing documentation ](https://github.ibm.com/event-integration/event-automation-docs/pull/1272/files#diff-e3da940789e2540c577724554c9e32f8d69cd7e48ffbc5a265c649172a6c2c56R138) 👍

Reply:
(and, even more importantly, in line with the Flink sample as installed with the operator in Flink v1.2.3)